theme_kiddo/static/src/scss/primary_variables.scss

```diff
@@ -184,6 +181,5 @@ $o-gray-color-palettes-compatibility-indexes: (
 // Shapes
 //------------------------------------------------------------------------------
 
-$o-bg-shapes: change-shape-colors-mapping('web_editor', 'Origins/14_001', (3: 2, 4: rgba(0, 0, 0, 0)));
```

Comment:
Can't remove this, unnecessary break on migration (you can add a comment to say it is kept for compatibility maybe and we'll somehow deal with them in ten years :shrug: ).